### DISCUSSION TITLE
Feature Request: "Fan Stop Protection"

### DISCUSSION THREAD

**Discussion by azidman** [Rem0o/FanControl.Releases#2487] [Ideas]
To guarantee that the fan is running FanControl should get a "Fan Stop Protection" option.
If a Fan has 0 RPM, FanControl gives it so much power, till it has a RPM value.
Then it switches back down to the configured settings.

**Comment by Rem0o** [comment]
That's what start/stop percent are for.
![image](https://github.com/Rem0o/FanControl.Releases/assets/14118956/c56e2916-b8f6-4c7f-b064-59d98737e001)

**Reply by Rem0o** [reply]
Just set start% to be like 80%, and that will be the kickstart value if it stops spinning.

**Reply by azidman** [reply]
Got it. Thx a lot. :)
